MonitoringSystem/Client/MonitoringReporter.py

```diff
@@ -33,36 +33,30 @@ class MonitoringReporter( object ):
   :param threading.RLock __documentLock: is used to lock the local store when it is being modified.
   :param __documents: contains the recods which will be inserted to the db\
   :type __documents: python:list
-  :param bool __mq: we can use MQ if it is available... By default it is not allowed.
   :param str __monitoringType: type of the records which will be inserted to the db. For example: WMSHistory.
-  :param object __mqProducer: publisher used to publish the records to the MQ.
-  """
+    """
 
   def __init__( self, monitoringType = '' ):
 
     self.__maxRecordsInABundle = 5000
     self.__documentLock = threading.RLock()
     self.__documents = []
-    self.__mq = False
     self.__monitoringType = None
-    
-    #check the existence of a MQ
-    retVal = getMQParamsFromCS ( "Monitoring::Queue::%s" % monitoringType )
-    if retVal['OK']:
-      result = createProducer( monitoringType )
-      if not result['OK']:
-        gLogger.warn( "Fail to create Producer:", result['Message'] )
-      else:
-        self.__mqProducer = result['Value']
-        self.__mq = True
-
+   
     self.__monitoringType = monitoringType
 
   def processRecords( self ):
     """
     It consumes all messaged from the MQ (these are failover messages). In case of failure, the messages
     will be inserted to the MQ again.
     """
+    retVal = monitoringDB.pingDB()  # if the db is not accessible, the records will be not processed from MQ
+    if retVal['OK']:
+      if not retVal['Value']:  # false if we can not connect to the db
+        return retVal
```

Comment:
there is two cases:
1.  can not connect to the services: monitoringDB.pingDB() return S_ERROR then this error message will be returned
2. retVal['OK']: managed to connect to the service or db, but the ping does not work in that case result['Value'] = False and continue processing the data

We will continue processing the records anyway so the return value does not really matter. The question here is do we want to return and error message saying can not ping the cluster or not? I do not think it is important, because in that case we know there is a problem (from the cluster monitoring page...)
But I am not against to return S_ERROR.